src/keeper/contracts/royalties/RewardsDistributor.ts

```diff
@@ -0,0 +1,53 @@
+import ContractBase, { TxParameters } from '../ContractBase'
+import { InstantiableConfig } from '../../../Instantiable.abstract'
+import Account from '../../../nevermined/Account'
+import { didZeroX, zeroX } from '../../../utils'
+import BigNumber from 'bignumber.js'
+
+export class RewardsDistributor extends ContractBase {
+    public static async getInstance(config: InstantiableConfig): Promise<RewardsDistributor> {
+        try {
+            const token: RewardsDistributor = new RewardsDistributor('RewardsDistributor')
+            await token.init(config, true)
+            return token
+        } catch (e) {
+            
```

Comment:
in this case we should either raise the exception or at least add a logger message